src/main/java/core/basesyntax/storage/Storage.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.storage;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static final Map<String, Integer> FRUIT_STORAGE = new HashMap<>();
+
+    public Map<String, Integer> getFruitStorage() {
+        return FRUIT_STORAGE;
+    }
```

Comment:
The `getFruitStorage` method returns a direct reference to the static storage map. Consider returning an unmodifiable view or a copy of the map to prevent unintended external modifications.